src/pyH2A/Plugins/Photovoltaic_Plugin.py

```diff
@@ -105,27 +122,86 @@ def calculate_H2_production(self, dcf):
 		yearly_data = []
 
```

Comment:
Here osmosis value for the first year when the electrolyzer would run fully. For this a separate function below is listed (so the code is cleaner). 100% auslastung von electrolyzer